### DISCUSSION TITLE
[Weekly Review Report]: Feb 10 - 16

### DISCUSSION THREAD

**Discussion by AlexanderPico** [wikipathways/wikipathways-help#107] [curation-reports]
### Recent changes

 * 47 edits reviewed 
   * Mostly refinements to layout, casing, typos, and ontology terms. 
 
_**Thanks, @Eweitz for >200 edits in past couple weeks; ~600 since 1-Jan!**_

### Under construction

 * Completed [WP5442](https://classic.wikipathways.org/index.php/Pathway:WP5442)
 * Approved [WP5424](https://classic.wikipathways.org/index.php/Pathway:WP5424)

### New pathways

 * [WP5442](https://classic.wikipathways.org/index.php/Pathway:WP5442) _**Thanks, @andrawaag!**_ 

### Deleted Pathways

 * 1 test pathway from January 
 * 4 AOPs requested by Marvin

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

### Additional notes

* Added a **Request Deletion** tag to the only remaining "stub" pathway: [WP2512](https://classic.wikipathways.org/index.php/Pathway:WP2512). If no one objects, then we can delete it in 30 days.
* Search tested and working

_**@Tabbassidaloii, you're up next!**_

**Comment by danidi** [comment]
I don't have Pathvisio at hand to change it at the moment, but in WP5442 it would be good to use Uniprot-Trembl instead of Swissprot as database (it will not be mapped to other data sources otherwise). Also Ensembl Human doesn't seem to get any mappings, I guess it should be Ensembl instead.
